docs/packages/pkg/sds.rst

```diff
@@ -0,0 +1,21 @@
+.. spelling::
+
+    sds
+
+.. index:: containers ; sds
+
+.. _pkg.sds:
+
+sds
+===
+
+-  `Official <https://github.com/antirez/sds>`__
+-  `Hunterized <https://github.com/hunter-packages/sds>`__
```

Comment:
I realize I should have done this upfront, but I still need a fork of this package.

Reply:
Added: https://github.com/hunter-packages/sds